lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1594,11 +1594,19 @@ def cmd_db_nmap(*args)
       print_status("Usage: db_nmap [nmap options]")
       return
     end
-
+    
```

Comment:
I'm planning on removing this line, but I was hoping to do so with any other changes that may be necessary.  As such, I'll keep this on hold until I receive further notice on changes for this PR.  Thanks!